apps/src/calc/calc.js

```diff
@@ -716,6 +724,7 @@ function isPreAnimationFailure(testResult) {
     testResult === TestResults.EMPTY_FUNCTIONAL_BLOCK ||
     testResult === TestResults.EXTRA_TOP_BLOCKS_FAIL ||
     testResult === TestResults.EXAMPLE_FAILED ||
+    testResult === TestResults.APP_SPECIFIC_FAIL || // TODO(bcjordan): is this wrong?
```

Comment:
Ooh, actually upon removing this line addition it still works fine (it just shows the equation below the error message, which is how it's displayed on non-project levels anyway). Seems safer to not change this logic.